Was just looking into this too. Please mind that `npm run build` presently errors, at least on my install:
```
src/lib/assets.ts:16:60 - error TS2322: Type 'AssetQuery' is not assignable to type 'Record<string, unknown>'.
  Index signature for type 'string' is missing in type 'AssetQuery'.

16   const { data, errors } = await request({ path: "/asset", params })
                                                              ~~~~~~

  src/utils.ts:17:3
    17   params?: Record<string, unknown>;
         ~~~~~~
    The expected type comes from property 'params' which is declared here on type 'RequestPayload'

src/lib/assets.ts:30:64 - error TS2322: Type 'MapPath & VersionQuery & Pick<AssetQuery, "format">' is not assignable to type 'Record<string, unknown> | undefined'.
  Type 'MapPath & VersionQuery & Pick<AssetQuery, "format">' is not assignable to type 'Record<string, unknown>'.
    Index signature for type 'string' is missing in type 'MapPath & VersionQuery & Pick<AssetQuery, "format">'.

30   const { data, errors } = await request({ path: "/asset/map", params })
                                                                  ~~~~~~

  src/utils.ts:17:3
    17   params?: Record<string, unknown>;
         ~~~~~~
    The expected type comes from property 'params' which is declared here on type 'RequestPayload'


Found 2 errors in the same file, starting at: src/lib/assets.ts:16
```

--- a/src/lib/assets.ts
+++ b/src/lib/assets.ts
@@ -13,7 +13,7 @@ export class Assets {
    * @see https://v2.xivapi.com/api/docs#tag/assets/get/asset
    */
 	async get(params: Models.AssetQuery): Promise<Buffer> {
-		const { data, errors } = await request({ path: "/asset", params })
+		const { data, errors } = await request({ path: "/asset", params: params as unknown as Record<string, unknown> })
 		if (errors) throw new CustomError(errors[0].message)
 		return data as Buffer
 	}
@@ -27,7 +27,7 @@ export class Assets {
 	async map(
 		params: Models.MapPath & Models.VersionQuery & Pick<Models.AssetQuery, "format">
 	): Promise<Buffer> {
-		const { data, errors } = await request({ path: "/asset/map", params })
+		const { data, errors } = await request({ path: "/asset/map", params: params as unknown as Record<string, unknown> })
 		if (errors) throw new CustomError(errors[0].message)
 		return data as Buffer
 	}

